scripts/dataset_processing/tts/preprocess_audio.py

```diff
@@ -123,12 +129,18 @@ def _process_entry(
     overwrite_audio: bool,
     audio_trimmer: AudioTrimmer,
     output_sample_rate: int,
+    output_format: str,
     volume_level: float,
 ) -> Tuple[dict, float, float]:
     audio_filepath = Path(entry["audio_filepath"])
 
     audio_path, audio_path_rel = get_abs_rel_paths(input_path=audio_filepath, base_path=input_audio_dir)
+
+    if not output_format:
+        output_format = audio_path.suffix
+
```

Comment:
we need a safeguard to cover only supported audio formats, otherwise, raises a `ValueError`. Are the formats supported the same to Librosa.load() or torchaudio.load()?

Reply:
I added a check. I am not sure if torchaudio and librosa support the same audio formats.